### DISCUSSION TITLE
Example of profiles?

### DISCUSSION THREAD

**Discussion by lbeltrame** [nirenjan/x52pro-linux#37] [Q&A] [answered]
The configuration file hints at a profile directory, however it is not clear how profiles work (if they are implemented) and an example of them.

**Comment by nirenjan** [comment, marked as answer]
Not yet implemented - it was my intent to implement something similar to the way the Saitek profile editor works on Windows, but that's still something still in the early concept phase.

**Reply by lbeltrame** [reply]
It might be useful to add a comment in the configuration file or in the README to that extent, IMO.

**Reply by nirenjan** [reply]
Done. See 09273ae894a475d3ab7fa2dc5f89bc2ce36ec392
